webhooks/start/package.json

```diff
@@ -14,6 +14,7 @@
     "body-parser": "^1.20.0",
     "dotenv": "^16.0.0",
     "express": "^4.18.1",
+    "fs.promises": "^0.1.2",
```

Comment:
Thank you for the PR!

So, fs.promises appears to be this package: https://github.com/cravler/fs.promises 

whereas this repo is using the NodeJS Promises API 
https://nodejs.org/api/fs.html#promises-api

They are different, and this project does not require the fs.promises package. I was able to get the project to run as-is without making these changes. As fs.promises is not the mainline way to use the promises API and does not appear to be maintained we do not want to create a dependency on it. 
Based on your comment it appears that you are using a different version of node than the one recommended in the readme -- perhaps try that one and see if that works, and/or provide the error message you're getting?

Reply:
I am using Node v12.18.1 unfortunately, I did not want to upgrade since the project I am on is what version the company is using.  

I just wanted to see how this works, based on the YT video.

Reply:
Hmmm... yeah, I don't think this will run on Node 12. Have you considered using a tool like [nvm](https://github.com/nvm-sh/nvm)? That would allow you to easily install and switch between different node versions so you can try projects like this while still keeping the older version around for your company's project.

Reply:
To close the loop, I confirmed that the syntax for importing fs/promises changed between Node 12 and Node 14, which explains the issue. https://github.com/nodejs/node/issues/35740

Going to close this PR since this code should work correctly on more recent versions of node.